### ISSUE TITLE
Prevent setting initial scroll position?

### ISSUE THREAD

**Issue by marjan-georgiev** [staskolukasz/ngx-infinite-scroller#7]
Hi there, nice directive!

Was wondering if there is a way to prevent calling `setInitialScrollPosition`? 
https://github.com/staskolukasz/ngx-infinite-scroller/blob/master/src/app/scrolling-strategy/scrolling-to-both.ts#L53

I have a list of chat messages (newest on bottom), and would like to be able to scroll to bottom immediately after rendering the component and populating it with the latest batch of messages. I use the  scroll-to-both strategy, and it is automatically positioning the scroll at the middle after a second (scrollDebounceTimeAfterDOMMutationOnInit) right now, overriding my scroll position.

Would be nice if we could pass an input to the directive to disable initial scroll position.

**Comment by staskolukasz** [comment]
Hello!
Thank you for your feedback! I really appreciate that!

I like the idea of setting initial value of scroll position via input value. I implemented this feature and to be honest, I found it very enjoyable.
I've also upgraded the library to use Angular 7 package.

I'm not sure if you still need this new feature, but maybe somebody else will need it.
Best Regards

https://github.com/staskolukasz/ngx-infinite-scroller/pull/8
